OpenRA.Mods.Common/Traits/Buildings/ProductionAirdrop.cs

```diff
@@ -44,6 +43,83 @@ class ProductionAirdrop : Production
 		public ProductionAirdrop(ActorInitializer init, ProductionAirdropInfo info)
 			: base(init, info) { }
 
+		public override bool Produce(Actor self, IEnumerable<ActorInfo> producees, string productionType, TypeDictionary inits)
+		{
+			if (IsTraitDisabled || IsTraitPaused)
```

Comment:
Duplicating all of the body here isn't great. This is a case where it does make sense for the single-actor version to call this.

Reply:
Removed the duplication by a fair bit.